tutorials/howto/howto_new_profile_version.md

```diff
@@ -0,0 +1,177 @@
+  keywords: "schema.org, markup, structured data, bioschemas profile"
+  license: CC-BY 4.0
+  version: 1.0
+---
+
+## 1. Prepare your working environment
+
+To create and publish a new draft profile you need access to a couple of Bioschemas resources, here we list them all together with the main role they play in the whole process. We will explain how to use them and what to do with them in later steps.
+
+* Bioschemas specifications [GDrive folder](https://drive.google.com/drive/folders/0B8yXU9SkT3ftaWJtTGYyTTJjck0): to create the spreadsheet corresponding to the new draft profile, you need editing rights
+* [GOweb repository](https://github.com/BioSchemas/bioschemas-goweb): to move from the spreadsheet to the website
+* [Website repository](https://github.com/BioSchemas/bioschemas.github.io): to publish the new draft profile, you need editing rights
+* [Specifications repository](https://github.com/BioSchemas/specifications): to add compliant examples regarding the new draft profile, you need editing rights
+
+If you do not have the right permissions to edit the mentioned resources, send an email to the [mailing list](mailto:public-bioschemas@w3.org) explaining why you should have access and what Bioschemas group you belong to. Once your case has been validated, you will get an invitation to join the GDrive and the repositories.
+
+From now on, we will use ScholarlyArticle as an example to create and publish a new draft profile.
+
+## 2. Create a new profile crosswalk
+
+A profile crosswalk is the cornerstone of Bioschemas profile development. It is a spreadsheet with the Schema.org fields to be considered together with cardinality, marginality, suggested controlled vocabularies and examples. It is not a regular spreadsheet but one with a template and some predifined functionalities. It is then translated into strcutured data so that the Bioschemas web site is automatically updated. 
+
+A new profile is commonly based on the previous one so the easiest way to start is from the previous version.
+
+* Go to the [GDrive](https://drive.google.com/drive/folders/0B8yXU9SkT3ftaWJtTGYyTTJjck0) and navigate to the folder [Specifications](https://drive.google.com/drive/folders/0Bw_p-HKWUjHoNThZOWNKbGhOODg)
+* Locate and open the folder corresponding to the profile you want to update, in our example it is the [ScholarlyArticle](https://drive.google.com/drive/folders/1CgEyta4d7vFfYT7r7w9HtLZ4kAidDmJI) folder
+* Create a copy of the latest ScholarlyArticle Mapping file. At the time of writing, it was the ScholarlyArticle Mapping 0.2-DRAFT
+* Rename the copy so it reflects the new draft version, in this case it would be ScholarlyArticle Mapping 0.3-DRAFT
+
+### 2.1. The Specification Info tab
+
+In this tab, you need to update the "Description" column. In most cases the actual description of the profile will remain the same but the "Summary of Changes" will change
+* List all the changes in this new draft profile
```

Comment:
State how this should be done, i.e. by adding `<h3>Summary of Changes</h3><ul><li>...`

Reply:
done